src/analysis/modifications/optimizer.jl

```diff
@@ -0,0 +1,36 @@
+
+"""
+$(TYPEDSIGNATURES)
+
+Change the objective sense of optimization. Possible arguments are
+`JuMP.MAX_SENSE` and `JuMP.MIN_SENSE`.
+"""
+modify_sense(objective_sense) =
```

Comment:
we don't need a modification nomenclature here so maybe we can even have the same names as JuMP or so, to prevent chaos.

Too bad julia doesn't have partial application, that would be exactly it

Reply:
aahhh

Reply:
okay